fs-integration-test-suite/accelerator-test-framework/src/main/groovy/org/wso2/financial/services/accelerator/test/framework/constant/AccountsRequestPayloads.groovy

```diff
@@ -45,8 +45,7 @@ class AccountsRequestPayloads {
             ConnectorTestConstants.READ_ACCOUNTS_BASIC,
             ConnectorTestConstants.READ_ACCOUNTS_DETAIL,
             ConnectorTestConstants.READ_BALANCES,
-            ConnectorTestConstants.READ_TRANSACTIONS_DETAIL,
-            ConnectorTestConstants.READ_TRANSACTIONS_DEBITS
+            ConnectorTestConstants.READ_TRANSACTIONS_DETAIL
```

Comment:
READ_TRANSACTIONS_DEBITS should come with READ_TRANSACTIONS_DETAIL. Let's not remove it.

Reply:
in Accelerator, we don't support that